net-kit/autogen.kit.d/base.yml

```diff
@@ -2627,4 +2627,159 @@ gitlab_freedesktop_dirlisting:
               rmdir "${ED}"/var{/lib{/NetworkManager,},} || die
             }
 
+libcupsfilters:
+  generator: builtin-github
+  template:
+    engine: helm
+
+  defaults:
+    files_dir: "{{ .Values.pn }}"
+    template: ../../templates/simple.tmpl
+    category: net-print
+    github:
+      user: OpenPrinting
+      repo: libcupsfilters
+    assets:
+      - name: '{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+        url: 'https://github.com/OpenPrinting/{{ .Values.pn }}/releases/download/{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+    vars:
+      license: Apache-2.0
+      slot: '0'
+      homepage: https://github.com/OpenPrinting/libcupsfilters
+      inherit:
+        - libtool
+      iuse: dbus exif jpeg +poppler +postscript png test tiff
+      restrict: '!test? ( test )'
```

Comment:
Also here `test` could be removed and set to `false` statically in the configure